src/index.ts

```diff
@@ -3,11 +3,27 @@ import * as d from './declarations';
 import { loadDiagnostic } from './diagnostics';
 import { createResultsId, getRenderOptions, usePlugin } from './util';
 
+/**
+ * The entrypoint of the Stencil Sass plugin
+ *
+ * This function creates & configures the plugin to be used by consuming Stencil projects
+ *
+ * @param opts options to configure the plugin
+ * @return the configured plugin
+ */
 export function sass(opts: d.PluginOptions = {}): d.Plugin {
```

Comment:
Am I right that this is the function people will import into their Stencil config if they want to use sass? If that's right, it might be good to adorn this jsdoc with just a few more helpful things, like maybe just a link to the README (https://github.com/ionic-team/stencil-sass) and possibly to the plugin documentation (https://stenciljs.com/docs/plugins) although the latter is a bit spare at the moment. I'm just thinking that that way when you hover over the symbol in your editor you get some sort of a friendly message in the JSDoc.

Reply:
👍 done in https://github.com/ionic-team/stencil-sass/pull/225/commits/2f22174b0dbf47cd89836fdbb69bce1aca133404